doc/api/worker_threads.md

```diff
@@ -1191,9 +1191,10 @@ changes:
       used for generated code.
     * `stackSizeMb` {number} The default maximum stack size for the thread.
       Small values may lead to unusable Worker instances. **Default:** `4`.
-  * `name` {string} An optional `name` to be appended to the worker title
-    for debugging/identification purposes, making the final title as
-    `[worker ${id}] ${name}`. **Default:** `''`.
+  * `name` {string} An optional `name` to be replaced in the thread name
+    and to the worker title for debugging/identification purposes,
+    making the final title as `[worker ${id}] ${name}`.
+    **Default:** `'WorkerThread'`.
```

Comment:
There are size limitations to the name, correct? Those should be documented here.

Reply:
Yes, I'll adjust that

Reply:
The only platform I'm aware has an encoding limitation is `illumos`. It's being covered by the uv test: https://github.com/libuv/libuv/commit/61c966cf0b4b6b5fc44d5ab3678cf5d0caef249d#diff-39afa11511d9f371ce2e3ecc18ddbec086588e206292c9b07de2e84fd062f85dR115